src/main/scala/cromwell/engine/backend/jes/Run.scala

```diff
@@ -88,11 +90,42 @@ object Run  {
   trait TerminalRunStatus extends RunStatus
   case object Initializing extends RunStatus
   case object Running extends RunStatus
-  case object Success extends TerminalRunStatus
+  case class Success(events: Seq[ExecutionEventEntry]) extends TerminalRunStatus
   final case class Failed(errorCode: Int, errorMessage: String) extends TerminalRunStatus {
     // Don't want to include errorMessage or code in the snappy status toString:
     override def toString = "Failed"
   }
+
+  // An event with a startTime timestamp
+  private case class EventStartTime(name: String, timestamp: DateTime)
+
+  def getEventList(op: Operation): Seq[ExecutionEventEntry] = {
+
+    val starterEvents = eventIfExists("createTime", op, "chilling out, maxing") ++ eventIfExists("startTime", op, "relaxing all cool")
+
+    val eventsList: Seq[EventStartTime] = if (op.getMetadata.containsKey("events")) {
+      op.getMetadata.get("events").asInstanceOf[java.util.ArrayList[AnyRef]].asScala map { x =>
+        val entry = x.asInstanceOf[ArrayMap[String, String]]
+        EventStartTime(entry.get("description"), DateTime.parse(entry.get("startTime")))
+      } toSeq
+    } else Seq.empty
+
+    // The final event is only used as the book-end for the final pairing (see below) so the name is never actually used...
+    // ... which is rather a pity actually - it's a jolly good name.
+    val finaleEvents = eventIfExists("endTime", op, "cromwell poll interval") ++ Seq(
+      EventStartTime("The Queen flying around with a jet-pack, with Winston Churchill cheering and waving a huge Union Jack in the background", DateTime.now))
```

Comment:
we should start a betting pool as to when this will first be surfaced to a real user :smile: